stories/SQForm.stories.js

```diff
@@ -91,22 +91,22 @@ const MOCK_FORM_FOR_MULTISELECT = {
 };
 
 const MOCK_FRIENDS_OPTIONS = [
-  {label: 'Joe', value: 'Joe'},
-  {label: 'Jane', value: 'Jane'},
-  {label: 'Jack', value: 'Jack'},
-  {label: 'Jill', value: 'Jill'},
-  {label: 'John', value: 'John'},
-  {label: 'Jocelyn', value: 'Jocelyn'},
-  {label: 'Jacob', value: 'Jacob'},
-  {label: 'Jackson', value: 'Jackson'},
-  {label: 'Josh', value: 'Josh'},
-  {label: 'Joseph', value: 'Joseph'},
-  {label: 'Jeremy', value: 'Jeremy'},
-  {label: 'Joel', value: 'Joel'},
-  {label: 'Jonah', value: 'Jonah'},
-  {label: 'Judah', value: 'Judah'},
-  {label: 'Jimmy', value: 'Jimmy'},
-  {label: 'Jessica', value: 'Jessica'}
+  {label: 'Joe', value: random(10 + Math.ceil(Math.random() * 20))},
```

Comment:
Put in random chars to simulate most of our data sets where the value is gonna be an ID of some kind and the label is going to be some user friendly string value.